### ISSUE TITLE
How do I use `getObjectStream` with a CRD?

### ISSUE THREAD

**Issue by craicoverflow** [godaddy/kubernetes-client#517]
I've followed the documentation to move from the deprecated `getStream` to `getObjectStream`. For example, the following works fine:

```js
const events = await client.api.v1.watch.namespaces(namespace).pods.getObjectStream()
```

But I cannot use `getObjectStream` with a CRD. 

```js
const stream = await kubeclient.apis['test.api.group'].v1alpha1.watch
      .namespaces(namespace)
      .apples.getStream();
```

I am getting the deprecation warning below, so I assume `getObjectStream` should be used. Could you please point me to an example of how to do this?
```
kubernetes-client deprecated .getStream see https://github.com/godaddy/kubernetes-client/blob/master/merging-with-kubernetes.md server/appServices.js:67:22
```

I checked the documentation, [using-crds.js](https://github.com/godaddy/kubernetes-client/blob/master/examples/using-crds.js) and [watch.js](https://github.com/godaddy/kubernetes-client/blob/master/examples/watch.js) do not provide examples for this scenario.

**Comment by silasbw** [comment]
included fix in `kubernetes-client@8.3.3`

**Comment by huineng** [comment]
i still have this, even in version 9 (for CRD)

```
kubernetes-client deprecated .getStream use .getObjectStream, see https://github.com/godaddy/kubernetes-client/blob/master/merging-with-kubernetes.md 
```
Property 'getObjectStream' does not exist on type 'ApiV1WatchEvents'.ts(2339)
